spec/rubocop/cop/style/array_join_spec.rb

```diff
@@ -11,6 +11,13 @@
     expect(cop.offenses.size).to eq(1)
   end
 
+  it "autocorrects '*' to 'join'" do
+    corrected =
```

Comment:
You have a few more examples in the PR description. Maybe you should add some of them to the spec.